trashradar-api/complaints/models.py

```diff
@@ -0,0 +1,13 @@
+from django.db import models
+
+
+class Entity(models.Model):
+    class Meta:
+        verbose_name = 'Entity'
+        verbose_name_plural = 'Entities'
+
+    name = models.CharField(max_length=200)
+    twitter = models.CharField(max_length=200)
+    phone = models.CharField(blank=True, max_length=20, null=True)
+    message = models.TextField()
```

Comment:
Is the entity every complaint message? Or the organization that should pick up the trash?

I see this as two tables:
- Let's say `entity` with name, twitter, phone, etc.
- And, `message` with content, timestamp, author, location, picture and a FK to the `entity`.